src/daily_topic_newsletter.py

```
@@ -0,0 +1,160 @@
+    def _send_email(self, html: str) -> None:
+        message = Mail(
+            from_email="news@example.com",
+            to_emails="you@example.com",
+            subject=f"Daily {self.topic} Newsletter",
+            html_content=html,
+        )
+        sg = SendGridAPIClient(SENDGRID_API_KEY)
+        sg.send(message)
+
+    # ------------------------------------------------------------------
+    def run_once(self) -> None:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)